scripts/verify-bun-review-artifacts.py

```
@@ -2079,17 +2079,72 @@ def require_github_comment(comment: dict, index: int) -> None:
+    if "\r" in text:
+        fail(f"github review comment {index} suggestion contains carriage returns")
+    lines = text.split("\n")
+    if len(lines) > GITHUB_SUGGESTION_MAX_LINES:
+        fail(
+            f"github review comment {index} suggestion exceeds "
+            f"{GITHUB_SUGGESTION_MAX_LINES} lines"
+        )
+    for line in lines:
+        if line.startswith(("@@ ", "--- ", "+++ ", "diff --git ")):
+            fail(f"github review comment {index} suggestion contains unified diff markers")
+        stripped = line.strip().lstrip("//").lstrip("#").lstrip("/*").rstrip("*/").strip()
```

Ruff (0.16.1)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)